hazelcast/src/test/java/com/hazelcast/internal/management/ExecuteScriptRequestTest.java

```diff
@@ -61,71 +62,68 @@ public void setUp() {
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
         HazelcastInstance[] instances = factory.newInstances();
 
-        cluster = instances[0].getCluster();
-        managementCenterService = getNode(instances[0]).getManagementCenterService();
-
-        bindings.put("key", "value");
+        Node node = getNode(instances[0]);
+        nodeAddressWithBrackets = node.getThisAddress().toString();
+        nodeAddressWithoutBrackets = node.getThisAddress().getHost() + ":" + node.getThisAddress().getPort();
+        managementCenterService = node.getManagementCenterService();
     }
 
     @Test
     public void testExecuteScriptRequest() throws Exception {
-        ExecuteScriptRequest request = new ExecuteScriptRequest("print('test');", "JavaScript", false, bindings);
-
-        JsonObject jsonObject = new JsonObject();
-        request.writeResponse(managementCenterService, jsonObject);
-
-        JsonObject result = (JsonObject) jsonObject.get("result");
-        String response = getString(result, "scriptResult");
-        assertEquals("", response.trim());
-    }
-
-    @Test
-    public void testExecuteScriptRequest_whenTargetAllMembers() throws Exception {
-        ExecuteScriptRequest request = new ExecuteScriptRequest("print('test');", "JavaScript", true, null);
+        ExecuteScriptRequest request = new ExecuteScriptRequest("print('test');", "JavaScript",
+                singleton(nodeAddressWithoutBrackets));
 
         JsonObject jsonObject = new JsonObject();
         request.writeResponse(managementCenterService, jsonObject);
 
         JsonObject result = (JsonObject) jsonObject.get("result");
-        String response = getString(result, "scriptResult");
-        assertEquals("error\nerror\n", response);
+        JsonObject json = getObject(result, nodeAddressWithBrackets);
+        assertTrue(getBoolean(json, "success"));
+        assertEquals("error\n", getString(json, "result"));
+        assertEquals("error\n", getString(result, "scriptResult"));
```

Comment:
Couldn't understand this one. Other tests in this class don't compare the output messages as a whole, they look for substrings or check if an empty response is returned.

Reply:
I just mean that this response ends with a `\n` while the other ones don't. Why would just the `"error"` message end with `\n`? It feels odd.

Reply:
I guess it was done so because the original developers preferred to display the message on MC as is , so they have added the newline to the message on the cluster side. For example, here we also see the newlines being added to the messages' end: https://github.com/hazelcast/hazelcast/blob/1f67f86f41115183eb1113943d29d129d776aa26/hazelcast/src/main/java/com/hazelcast/internal/management/request/ExecuteScriptRequest.java#L98